TRACING.md

```diff
@@ -116,6 +115,21 @@ Each trace mode is sub-divided into two different types:
 | isolated | The traces only apply to the agent; after the container has been destroyed, the first span will start at agent startup and the last at agent shutdown | Observing agent lifespan. | |
 | collated | In this mode, spans are associated with their `kata-runtime` initiated counterparts. | Understanding how the runtime calls the agent. | Requires runtime tracing to be enabled in `configuration.toml` (`enable_tracing=true`). |
 
+# Agent shutdown behaviour
+ 
+As shown in [Trace modes](#trace-modes) section, the different trace modes
+change how the VM is shutdown. In normal operation, the `kata-runtime` manages
+the VM shutdown. The **only** scenario when the agent itself is responsible for
```

Comment:
Suggested rewrite: 
.... The **only** scenario the agent itself is responsible for VM shutdown is if static tracing is enabled.